render-tests/geojson/clustered/style.json

```diff
@@ -0,0 +1,52 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "native": false,
+      "width": 512,
+      "height": 512
+    }
+  },
+  "center": [0, 0],
+  "zoom": 0,
+  "sources": {
+    "geojson": {
+      "type": "geojson",
+      "data": "local://data/places.geojson",
+      "cluster": true,
+      "clusterRadius": 200
+    }
+  },
+  "glyphs": "local://glyphs/{fontstack}/{range}.pbf",
+  "layers": [{
+    "id": "cluster",
+    "type": "circle",
+    "source": "geojson",
+    "filter": ["all", ["==", "cluster", true]],
+    "paint": {
+      "circle-color": "rgba(0, 200, 0, 1)",
+      "circle-radius": 20
+    }
+  }, {
+    "id": "cluster_label",
+    "type": "symbol",
+    "source": "geojson",
+    "filter": ["all", ["==", "cluster", true]],
+    "layout": {
+      "text-field": "{numPointsH}",
```

Comment:
`numPointsH` is pretty cryptic. How about `abbreviated_point_count`? (Underscores for consistency with mapbox-streets property name conventions.)